### ISSUE TITLE
Movies Update Issue

### ISSUE THREAD

**Issue by robertivanyi4** [popcorntime/popcorntime#1690]
Seems like Popcorn Time Movies won't get updated for a while (1-2 week(s)). I see the same movies in the same order in each view. Please advice me how to proceed. Thx. ('A Rebel Born' is last added movie.)

**Comment by dutchy1001** [comment]
Popcorn Time itself does not add any movies or series, there linked from YTS (YIFY Torrents) or similar torrent agents.
Every time you open PCT it will get updated from these torrents, if nothing has been added the list will not change then.

They have no control over what's added and not.

**Comment by Persei08** [comment]
@dutchy1001 In fact, torrents websites (YTS, Eztv, ...) are automatically requested/scrapped by popcorn-api servers many times per day to get new releases, and there are some additional routines to retrieve matching metadatas and covers from trakt.tv and TMDB. 
Then all those data (magnet links, sizes,... trakt.tv title, year, imdb id, synopsis, genres, ratings,... tmdb covers) are put together and saved into popcorn-api database. And PT client request the popcorn-api servers.

So yeah, PT team don't add content themselves, it's a automated process done by popcorn-api which automatically scrap data from yts and eztv.

But ~~sometimes~~ often, we take shortcuts when explaining to users why some contents are missing, because explaining what popcorn-api is and do isn't very useful when the reason is that's not available on the scrapped websites.

Here, the issue seem there is a problem since some weeks with the automated process which scrap data from yts.

**Comment by Persei08** [comment]
I digress a little bit from initial topic, but here an example of those data with the documentary "TPB AFK: The Pirate Bay - Away from Keyboard"... released under the Creative Commons license onto ThePirateBay (so this example should be ok to publish here)

```json
{
    "_id": "tt2608732",
    "imdb_id": "tt2608732",
    "title": "TPB AFK: The Pirate Bay - Away from Keyboard",
    "year": "2013",
    "synopsis": "TPB AFK is a documentary about three computer addicts who redefined the world of media distribution with their hobby homepage The Pirate Bay. How did Tiamo, a beer crazy hardware fanatic, Brokep a tree hugging eco activist and Anakata – a paranoid hacker libertarian – get the White House to threaten the Swedish government with trade sanctions? TPB AFK explores what Hollywood’s most hated pirates go through on a personal level.",
    "runtime": "82",
    "released": 1360281600,
    "trailer": "http://youtube.com/watch?v=6X8MU4iZwJQ",
    "certification": null,
    "torrents": {
        "en": {
            "1080p": {
                "url": "magnet:?xt=urn:btih:BB267C20DA25E8F3DC705027BE25FD6A87882121&tr=udp://glotorrents.pw:6969/announce&tr=udp://tracker.opentrackr.org:1337/announce&tr=udp://torrent.gresille.org:80/announce&tr=udp://tracker.openbittorrent.com:80&tr=udp://tracker.coppersurfer.tk:6969&tr=udp://tracker.leechers-paradise.org:6969&tr=udp://p4p.arenabg.ch:1337&tr=udp://tracker.internetwarriors.net:1337",
                "seed": 39,
                "peer": 5,
                "size": 1395864371,
                "filesize": "1.30 GB",
                "provider": "YTS"
            },
            "720p": {
                "provider": "YTS",
                "filesize": "699.68 MB",
                "size": 733667656,
                "peer": 0,
                "seed": 30,
                "url": "magnet:?xt=urn:btih:D9CEF4512E84CA5F2D824CBA0318363F92071DD2&tr=udp://glotorrents.pw:6969/announce&tr=udp://tracker.opentrackr.org:1337/announce&tr=udp://torrent.gresille.org:80/announce&tr=udp://tracker.openbittorrent.com:80&tr=udp://tracker.coppersurfer.tk:6969&tr=udp://tracker.leechers-paradise.org:6969&tr=udp://p4p.arenabg.ch:1337&tr=udp://tracker.internetwarriors.net:1337"
            }
        }
    },
    "genres": [
        "documentary"
    ],
    "images": {
        "poster": "http://image.tmdb.org/t/p/w500/3W2PYp6alnzCqhAzGt9Xs4iMkK4.jpg",
        "fanart": "http://image.tmdb.org/t/p/w500/vkB6HdWSlnbkD4TALfaqEsFv8yZ.jpg",
        "banner": "http://image.tmdb.org/t/p/w500/3W2PYp6alnzCqhAzGt9Xs4iMkK4.jpg"
    },
    "rating": {
        "percentage": 75,
        "watching": 0,
        "votes": 1137,
        "loved": 100,
        "hated": 100
    }
}
```

**Comment by robertivanyi4** [comment]
Thank you.

Üdvözlettel:
Iványi Róbert

2020. szept. 19. dátummal, 23:50 időpontban dutchy1001 <notifications@github.com> írta:

**Comment by robertivanyi4** [comment]
Thx. I see. Is there any estimation when this automated process, taking updates from yts, will get fixed?

Üdvözlettel:
Iványi Róbert

2020. szept. 20. dátummal, 0:55 időpontban Persei08 <notifications@github.com> írta:

**Comment by Persei08** [comment]
I can't really tell, but It already happened before and was fixed after some weeks.

**Comment by Alex1087** [comment]
Any updates for new movies? Almost a month waiting for new.

**Comment by robertivanyi4** [comment]
I'm afraid YTS got terminated, this is the reason...

Alex1087 <notifications@github.com> ezt írta (időpont: 2020. okt. 4., V,
7:38):

**Comment by kiriles90** [comment]
@robertivanyi4 YTS didnt get terminated.. their website and API work just fine. Its just Popcorn Time's API that is bugged.

**Comment by robertivanyi4** [comment]
I see. Thank you for information.

Üdvözlettel:
Iványi Róbert

2020. okt. 4. dátummal, 20:28 időpontban kiriles90 <notifications@github.com> írta:

**Comment by kiriles90** [comment]
No worries. Btw, until this is fixed, you can drag&drop or copy&paste magnet links from said YTS website (or from anywhere else) in Popcorn Time. The result will be the same.

Just.. if you use a VPN, make sure you have it on while you visit YTS (recent rumors about them handing over some users IPs and all..)

**Comment by dutchy1001** [comment]
It starts to be noticeable now that there are no movie/serie updates in the main tabs.

Using the torrent links from YTS is working fine, but I doubt many will use as the majority comes for the easy way.

**Comment by Persei08** [comment]
There isn't episodes updates, too ?

edit: yeah no nerver mind episodes are updated.

**Comment by dutchy1001** [comment]
I didn't check the series ( my mistake, sorry) as the series I have in my fav tab wont renew till next year

**Comment by michaelnagar** [comment]
Still no movies updates, Series are updating fine.
What would be the best URL to get those torrents you are referring to above and which can be "Dragged and Dropped into PCT"?
Thanks

**Comment by dutchy1001** [comment]
@michaelnagar 
You can use **[YTS](https://yts.mx/)** or any other torrent website as far I know.
Just make sure you use a VPN, I'm in the UK and without a VPN it will not show for me ( it could be my provider too).

**Comment by dutchy1001** [comment]
Is there any update to when this will be fixed?

Many PCT users are not that familiar how to add a torrent file/magnet link to PCT and just want the easy use.

**Comment by ivan1986** [comment]
Real some happen with api server
YTS works fine
my server has allmost all new english films from yts (last two only not exest - it's ok)
and only first url in api response - .sh

need check server
and maybe update server code to my? ;)

**Comment by Persei08** [comment]
@ivan1986 i've you tried popcorn-api, i'm not even able to use it. dependencies versions aren't good it need fixs here and there and maybe also require mongodb 3.2 which isn't available for ubuntu 18.04+.

was years last time i installed popcorn-api :/

**Comment by ivan1986** [comment]
@Persei08 i also try to extend api and after that full rewrite it to php

**Comment by Persei08** [comment]
@ivan1986  yeah, i start to wonder if it isn't better to replace popcorn-api.

**Comment by ivan1986** [comment]
@Persei08 I'll be happy to help deploy the server, clarify or modify something if you decide to use my code.

**Comment by Persei08** [comment]
@ivan1986 I was just speaking like that, I don't have responsibilities for that. I'm just collaborator, i can manage issues / PR here and launch build on CI. 

But I will try to run your api for personal usage :)

**Comment by michaelnagar** [comment]
Any good torrent site (with a good choice), still up/accessible in the UK (while on VPN) that you guys could suggest plz?
Thanks in advance

**Comment by dutchy1001** [comment]
Try **[YTS](https://yts.mx/)** (with VPN).
Any other torrent site should work from any site as far I know.

**Comment by macmsb** [comment]
If this is happening for everyone (?) I find it weird/worrying that only 25 people commented... Shouldn’t this be a top priority? Sincerely I feel embarrassed I only realized this a few days ago but am hoping this will be over soon.

**Comment by michaelnagar** [comment]
Still no movies update at all, its been ages now.
TV shows seem to update correctly

**Comment by michaelnagar** [comment]
> Try **[YTS](https://yts.mx/)** (with VPN).
> Any other torrent site should work from any site as far I know.

This has a very, very, very limited choice of films...
Any other ideas for torrents or better yet any idea when PopCorn-Time start working again! :-)

**Comment by ivan1986** [comment]
@michaelnagar use my fork for now - http://popcorn-ru.tk/
it work and has new english films

**Comment by michaelnagar** [comment]
Is anyone even looking into this issue???!!

> @michaelnagar use my fork for now - http://popcorn-ru.tk/
> it work and has new english films

Thanks mate!

**Comment by kiriles90** [comment]
@micaelnagar YTS is the **exclusive** movie provider for Popcorn Time, has been for years now... :)

**Comment by ivan1986** [comment]
Is someone have access to api server? Because real need fix it or update
for now 20361 films
maybe some issue with key of some api like tmdb or track and it return error for all films

**Comment by ivan1986** [comment]
and looks like we stop updating serials on main api
https://torrentfreak.com/major-torrent-site-eztv-has-domain-suspended-by-registry-201105/
on redit want mandalorian :smile:

**Comment by longnd** [comment]
> @michaelnagar use my fork for now - http://popcorn-ru.tk/
> it works and has new English films

@ivan1986 I guess to use your API server with the official Popcorn Time build for the desktop, we can set the Custom API Server in the advanced settings to `http://popcorn-ru.tk/movies/1?locale=en` ?
<img width="657" alt="Screen Shot 2020-11-07 at 15 56 58" src="https://user-images.githubusercontent.com/1896814/98437046-e2503500-2111-11eb-9b01-9796bc548787.png">

**Comment by ivan1986** [comment]
@longnd just http://popcorn-ru.tk/ - movies/<page> and shows/<page> will be add auto
locale send only my fork - default - english, but some other langs (not en and ru) useless now - very low content

**Comment by longnd** [comment]
@ivan1986 thank you, it works, except the Anime tab. I don't use it but just want to let you know. Btw, the response time of that API server is quite high, for example - when searching for a movie :D 
anw, I appreciate your work. Should you consider adding a line in the Readme of your repo https://github.com/popcorn-time-ru/popcorn-ru/ to indicate the way to use your API server, e.g. 
> Simply put `http://popcorn-ru.tk/` as the Custom API Server in the advanced settings if you use the official Popcorn Time app.

**Comment by br0k3nglass** [comment]
> @longnd just http://popcorn-ru.tk/ - movies/ and shows/ will be add auto
> locale send only my fork - default - english, but some other langs (not en and ru) useless now - very low content

Many thanks for providing this link, my wife will be very happy to have this updating again.

**Comment by macmsb** [comment]
@ivan1986 your link is working but app crashed at least twice after using it for a few seconds. Also screen in app flashes black every once in a while (like its opening up another window somewhere). Why do you think that is?

**Comment by ivan1986** [comment]
@macmsb not know - for me work perfect - last yesterday build work fast and not crash

**Comment by kiriles90** [comment]
@macmsb what build are you using? try updating to the latest one from ci.popcorntime.app

**Comment by confuzled-dev** [comment]
can we get an official solution instead of a users fork. This seems like a very odd thing to not be fixed for this long.

**Comment by kiriles90** [comment]
@dougshell you can use @ivan1986 API with the official app. Just set the 'Custom API Server' in the settings to http://popcorn-ru.tk/

**Comment by macmsb** [comment]
> @macmsb what build are you using? try updating to the latest one from ci.popcorntime.app

b2007344 macos

it is working, but again, seems dodgy that the app is flashing like that almost every time I go into a new screen

**Comment by kiriles90** [comment]
So 0.4.4 release version. You are using a very old build. Update to the latest one from ci.popcorntime.app like I said above. As for the .ru API change, nothing dodgy about it, you can check what it returns yourself by doing the calls manually in a browser..

**Comment by kiriles90** [comment]
Series are updating again. Editing the title.

**Comment by malikarshad786** [comment]
Can you update Queen's Gambit 6th episode. Its useless to watch.

**Comment by dutchy1001** [comment]
@malikarshad786 
Popcorn Time itself does not add any movies or series, there linked from YTS (YIFY Torrents) or similar torrent agents.

They have no control over what's added and not.

**Comment by gladykov** [comment]
This issue has a little bit more context about non working API: https://github.com/popcorn-official/popcorn-api/issues/142

**Comment by Asof1804** [comment]
It's the middle of December and I'm still having trouble finding new movies

**Comment by dutchy1001** [comment]
@Asof1804 
The issue is known and talked about. As far I know they're working towards a solution but it takes more time then expected.

**Comment by dutchy1001** [comment]
With the latest release of **[build 450](https://ci.popcorntime.app/job/Popcorn-Time-Desktop/lastStableBuild/)** you can add a custom server to "fetch" movies.
To do so you have to tick "show advance settings" and scroll down to "API server". Fill in at "Custom Movie Server" a known server (e.g. https://yts.mx/) and it should update the movie list.

PS.
Type the link instead of copy/paste as it will not save sometimes.

**Comment by Asof1804** [comment]
I wrote the link where you indicate me and I got a popcorn time error and
it did not load the movies or the series

[image: image.png]



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Remitente
notificado con
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
29/12/20
12:58:03

**Comment by kiriles90** [comment]
@Asof1804 means you didnt update to build 450 before doing that

**Comment by dutchy1001** [comment]
make sure you updated to the latest dev build.

**Comment by dutchy1001** [comment]
When turning off my VPN while using yts.mx I get this message

![Untitled](https://user-images.githubusercontent.com/5569980/103282647-5bf7f700-49ce-11eb-931e-20db90653ac0.png)

This means that my IPS is blocking YTS, don't think it can be more clear.

**Comment by sativ01** [comment]
for me the suggested `https://yts.mx/` didn't work, but if I drop `s` in `https` then it works. 
so Custom Movies Server set to `http://yts.mx/` worked for me, yet I assume it's not safe to use it.

anyone knows how to make it work with `https`?

Edit: this is indeed an IPS blocking `https://yts.mx/`, have to use VPN to make it work

**Comment by kiriles90** [comment]
@sativ01 thats very weird since yts automatically redirects you to https even if you use http. (try it in your browser with their website or API url). Either way, since the above it true you are fine with http too.

**Comment by dutchy1001** [comment]
both http and https are blocked by my ISP, as soon I turn my VPN on they both work.

**Comment by BGamboa13** [comment]
https://yts.mx/ does not filter new movies!!

**Comment by ivan1986** [comment]
@BGamboa13 of couse - it's not popcorn api - it's tracker api - it's not support full features

**Comment by kiriles90** [comment]
@BGamboa13 The ones YTS API shows are the actual new releases. You probably mean it doesnt have a 'Trending' filter but you can kind of simulate it by sorting by 'Trending' or 'Popularity' (they are the same, both Popularity) and search for 2020 (or 2014 if you want to see what was trending in 2014)

**Comment by hamzaathar** [comment]
Easy fix for it search a custom API server online and put it in the settings. The default API server is clearly not updating movies. I am using http://popcorn-ru.tk/

**Comment by amitayubasu** [comment]
> Easy fix for it search a custom API server online and put it in the settings. The default API server is clearly not updating movies. I am using http://popcorn-ru.tk/

How do I access these settings on Mac Catalina? It's not on the settings screen within popocorn time app

**Comment by dutchy1001** [comment]
There should be on top of the settings page a tic in "Show Advance Settings". This will allow you for more detailed settings.

**Comment by ghost** [comment]
Why isn't this issue fixed yet? Or why isn't there even a non-development release that makes a workaround possible?

Unrelated curiosity:
- Until https://github.com/popcorn-official/popcorn-desktop/projects/1 is fully addressed, won't there be a next stable release?
- Will future releases break bookmarks.db, settings.db etc.?

**Comment by dutchy1001** [comment]
>Why isn't this issue fixed yet?

There are some very sensitive issues that cant be updated due to some government regulations if you get my drift here.

>Will future releases break bookmarks.db, settings.db etc.?

With the newer dev build releases all settings are correctly brought over to the next version.
To get a dev build go **[here](https://ci.popcorntime.app/job/Popcorn-Time-Desktop/lastStableBuild/)**

**Comment by kiriles90** [comment]
@gnu1dea 

- No, that isnt a milestone.
- Current dev builds dont break anything so why would the release versions? (unless someone somehow reintroduces a relevant bug)

**Comment by dutchy1001** [comment]
I think the current "web" release (0.4.4) does not save the .db settings correctly. 
You need to export these in the settings before updating and re-add after an update from the dev builds. After you have a newer dev build it is fine and save your current .db settings.
I also had an issue that importing settings made PCT get stuck at loading unless you did it manual leaving the settings.db out (this doesn't happen with the newer dev builds either anymore).

**Comment by kiriles90** [comment]
> I also had an issue that importing settings made PCT get stuck at loading unless you did it manual leaving the settings.db out (this doesn't happen with the newer dev builds either anymore)

It was the 'UI scaling' and the old 'Big Picture' mode messing with each other. It was fixed a couple of builds later.

**Comment by ghost** [comment]
I don’t think I can continue to recommend PopcornTime.app to typical users if this isn’t fixed ‘proper’ in the installer by default without the user needing to go-to settings to change API.

**Comment by kiriles90** [comment]
@ReaLJohnBaker Too much trouble?

**Comment by ghost** [comment]
> @ReaLJohnBaker Too much trouble?

Unfortunately yes..

**Comment by kiriles90** [comment]
Without going into how easy it is to copy/paste a url in a textbox (...) or that in most apps you need to spend 5-10 minutes (if not a lot more) setting some things up in the settings, well.. then you can just recommend something easier to install and use to people. Popcorn Time is a free app, 'you wont get recommended' isnt much of a threat to anyone.

**Comment by sativ01** [comment]
Maybe @ReaLJohnBaker was looking for an easier option to select/switch a provider?
IMO could be nice if there was a dropdown with a list of supported providers, plus a user can add their own provider in settings. For example:
- Default
- YTS
- Popcorn-Ru
- My provider

And by this I can volunteer to add this feature if decided to move on with it.

------ off-topic
I mean it isn't hard to add a provider manually in settings, but to get to this solution I had to go to the github and find this particular issue. And before that I was downloading the app's every build for about a month, hoping to see new movies appearing. And I've been using PCT since 2015 🤷 

Although it is indeed rude to threat to the open source maintainers 🤦

**Comment by kiriles90** [comment]
@sativ01 Those textboxes are already dropdowns (`datalist`) but we cant add any urls in Popcorn Time's code. The official API isnt anywhere in Popcorn Time's code either. Its the _butter-settings-popcorntime.io_ module that needs to be edited.

**Comment by dutchy1001** [comment]
I understand you cant add servers for (see my comment above marked as Off-Topic) some reasons, But could it be like adding "custom 1, custom 2,  etc. .etc." and then fill it in yourself. Every time you add yourself a "custom" server a new empty "custom" field will be added at the bottom of that list.

**Comment by dutchy1001** [comment]
There is nothing wrong with the current system for adding servers and if the way it is now is future proof then that's the way to go.

**Comment by sativ01** [comment]
@kiriles90  I was thinking of adding a dropdown to the main window, e.g. like so (but proper design of course):
![image](https://user-images.githubusercontent.com/25383282/110604933-e7d58e80-8188-11eb-9cb2-a23bd2dbcfa2.png)

**Comment by sativ01** [comment]
@kiriles90
> Also why put it in the main window?
* For easier access

> You'll still have to restart the app for everything to work correctly
* could restart be triggered from the app?

> adding more stuff will only make the clutter and issue worse
* couldn't agree more! It is packed!

As I wrote previously, I'm fine with the way it is now, I've fixed it for myself by adding custom provider, but it was hard to find the correct solution. 
I'm simply thinking about the way to help inexperienced users to overcome the problem with movies not updating.

But if you think that's unnecessary, totally OK with me!

**Comment by ivan1986** [comment]
@kiriles90
> The official API url isnt anywhere in Popcorn Time's code either, that was the whole point with Butter etc. Its the butter-settings-popcorntime.io module that needs to be edited.

Maybe create new npm package butter-settings-popcorntime.app with new configs with working servers - as same as i do with my fork - https://github.com/popcorn-time-ru/butter-settings-popcorn

**Comment by kiriles90** [comment]
@sativ01 You are right in that. This should be marked clearer so users can find it without the need to ask someone. I still think its place is in the settings page though. Updating the butter-settings module to diffchecker.com/Uuerv5zj would help.

@ivan1986 @team-pct @Persei08

**Comment by ivan1986** [comment]
@kiriles90 I aggree - on settings page only - it's setup one time, but need set working defaults - users is fearful people

**Comment by Soldierkid** [comment]
dutchy anybody help me , or email me please shot_kall@hotmail.com so i aint watched movies for 14 months and now i got my laptop back last month but there was a update i guess and now my screen is black without my favorite likes movies i cant press any button im really sad, i watched you dutchy on other sites i saw you commenting helping people, how do i fix this problem ? my laptop is also full of space do i have to delete popcorn time and download it again and all my liked movies come back with the heart logo on popcorn time ??

**Comment by dutchy1001** [comment]
@Soldierkid 
I'll open a new issue (and close it strait away) as this goes off-topic in this tread.

**Comment by Soldierkid** [comment]
> @Soldierkid
> I'll open a new issue (and close it strait away) as this goes off-topic in this tread.

i dont know what you mean sir

**Comment by dutchy1001** [comment]
@Soldierkid 
Go to this link **[here](https://github.com/popcorn-official/popcorn-desktop/issues/2058)**

**Comment by Tylers-dad** [comment]
Non programmer and perplexed OAP here. Question. I have Popcorn both on my desktop PC (Win 10) and my Android tablet. Like others on here, my desktop version has not updated anything since 2020, but on my tablet, Popcorn is updating regularly with all the latest 2021 films. Am I correct in thinking this is because of the different operating systems?

**Comment by ivan1986** [comment]
@Tylers-dad looks like you right - on android you have other version which suported by other team

**Comment by Soldierkid** [comment]
What what

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by kiriles90** [comment]
Closing this since its an API issue and not much anyone here can do about it but leaving it pinned (as well as #2105 its pretty much the same issue and solutions) so everyone can find them until they are fixed or something else changes.

**Comment by ObscureCode** [comment]
> @michaelnagar use my fork for now - http://popcorn-ru.tk/
> it work and has new english films

Legend thank you!

**Comment by Soldierkid** [comment]
Hello I’ve got a important question
Get Outlook for iOS<https://aka.ms/o0ukef>

**Comment by Soldierkid** [comment]
Can you help me with my problem solve it for me
Get Outlook for iOS<https://aka.ms/o0ukef>

**Comment by gladykov** [comment]
> Can you help me with my problem solve it for me Get Outlook for iOS<https://aka.ms/o0ukef>

@Soldierkid Remove old, install new:
https://popcorn-time.ga/build/

**Comment by dutchy1001** [comment]
@Soldierkid stop spamming the same link. 
If you have a screenshot post that pic here but not links as that could be seen as malicious.

**Comment by Soldierkid** [comment]
Does that bring all my liked ones movies back the ones with the heart on it? My whole popcorn time is black if i unstall it and install it again is all my liked movies back ?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by dutchy1001** [comment]
If you can't see your settings tab to export  your database then go to this:
"C:\Users\YOURNAME\AppData\Local\Popcorn-Time\User Data\Default\data". 
Copy those 5 files to a safe location,
Uninstall the version you have,
Install a new version from the link given above by @gladykov,
If the Export Database was successful use the import function in the settings tab, if not, copy the files you put in a save location back to the same location you got the originals from.

Disclaimer:
If due to any means the database files have been corrupted you can try to add them 1 by 1 starting with "watched, movies, shows, bookmarks and last settings"
If that doesn't help there is noting you can do to get your liked/watched settings back.
